README.md

```diff
@@ -65,6 +65,8 @@ Pass the Webhook request body to `validatePayment`. Returns an error object cont
 validatePayment(responseBody)
 ```
 
+`Note`: The responseBody should be of type `x-www-form-urlencoded` as provided by SISP. The function itself handles the parse to JSON.
```

Comment:
This part isn't needed. The user doesn't need to know implementation details.
"The function itself handles the parse to JSON."